docs/deploy_features.md

```diff
@@ -17,35 +17,52 @@ You should typically enable these features before running `azd up`. Once you've
 * [Deploying with private endpoints](#deploying-with-private-endpoints)
 * [Using local parsers](#using-local-parsers)
 
-## Using GPT-4
+## Using GPT-4 or GPT-4o
 
-We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4, you can do so by following these steps:
+We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4 or GPT-4o, you can do so by following these steps:
 
 Execute the following commands inside your terminal:
 
 1. To set the name of the deployment, run this command with a new unique name.
 
+    For GPT-4:
     ```bash
     azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT chat4
     ```
+    For GPT-4o:
+    ```bash
+    azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT chat4o
+    ```
 
-1. To set the GPT model name to a **gpt-4** version from the [available models](https://learn.microsoft.com/azure/ai-services/openai/concepts/models), run this command with the appropriate gpt model name.
+1. To set the GPT model name to a **gpt-4** or **gpt-4o** version from the [available models](https://learn.microsoft.com/azure/ai-services/openai/concepts/models), run this command with the appropriate gpt model name.
 
+    For GPT-4:
     ```bash
     azd env set AZURE_OPENAI_CHATGPT_MODEL gpt-4
     ```
+    For GPT-4o:
+    ```bash
+    azd env set AZURE_OPENAI_CHATGPT_MODEL gpt-4o
+    ```
 
 1. To set the Azure OpenAI deployment capacity, run this command with the desired capacity.
 
     ```bash
     azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT_CAPACITY 10
     ```
+    For GPT-4o, you can skip this step if you prefer to use the default capacity.
```

Comment:
Hm, maybe I'm forgetting something, why is gpt-4o different from gpt-4 capacity-wise?